docs/Administrator Guide/SSL.md

```diff
@@ -60,6 +60,11 @@ the same resource, because either would be insecure.  Instead, any such "mixed
 mode" connections will be rejected by the TLS-using side, sacrificing
 availability to maintain security.
 
+One point to note is that the TLS certificate verification will fail if the
+machine times are not in sync with each other. Certificate verification
+depends on the time of the machine and if that is not found to be in sync then
+it is deemed to be an invalid certificate.
+
```

Comment:
Would it be useful to insert/link about how to ensure that the time on the machine instances are in sync? While this section focuses on the warning/troubleshooting aspect, the remedy might need to be linked as well.

Reply:
Ack. Will add a line about using ntpdate for syncing time.